### PR TITLE
Updated information on special flows.md

### PR DESCRIPTION
**Pull request by VidarHolm**

### CODE REVIEW

**Comment by VidarHolm** [comment]
Have added descriptions on flows and done adjustments to description on implementation. Have moved descriptions on the implementation of no dialog flow together with the other special flows.

File: vipps-login-api.md [hunk 1]
```diff
@@ -93,11 +93,31 @@ The user is then authenticated in browser and can provide consent if required. T
...
+If the user is on a mobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps app the user confirms the login and can choose whether to be rememberd in the browser for later logins. The standard flow after is that after confirming in the app the user needs to switch back to the Vipps page in the browser/app. On the Vipps landing page the user will finalise the autentication and provide consents if required. The user is then redirected back to the redirect URI provided by merchant (could be webpage or an app).
```
Comment (thormarius) [inline vipps-login-api.md:96]: 2 x same typo: rememberd => remembered

Comment (thormarius) [inline vipps-login-api.md:96]: The standard flow after is that after ?

File: vipps-login-api.md [hunk 2]
```diff
@@ -93,11 +93,31 @@ The user is then authenticated in browser and can provide consent if required. T
...
 
 ![Mobile flow with app-switch](images/Mobile_flow_with_partial-app_switch.png)
 
-We recommend that apps initiate Vipps login in a webview. SafariViewController and Chrome Custom Tabs are preferred as these webviews are able to utilize cookies stored in the user's browser.
+We recommend that apps initiate Vipps login in a webview. SafariViewController and Chrome Custom Tabs are preferred as these webviews are able to utilize cookies stored in the user's browser and is known to support required app switch to the Vipps app.
```
Comment (thormarius) [inline vipps-login-api.md:100]: and is known to support => and they are known to support

**Pull request merged**